beyond-essence/inventory/links.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: beyond-essence
+header_image: private/main.jpg
+title: Links
+background_image: index.jpg
+---
+
+Links are navigation elements.
+They are interactive and give users access to further content that is not directly accessible via the primary navigation.
+
+## Use cases
+
+In the Beyond cockpit, we provide various types of links:
```

Comment:
I would write something like:
> In the Beyond cockpit we provide two types of links:

Primary link
Default link

They vary in the form that they are used with icons or without. An icon before the link supports the content. A link without icon is used for self explained links.

Reply:
I've moved the "old" use cases to the "position" section and added the primary and secondary links as new use cases. 

When having a look at the primary/secondary issue again, I also figured out that I need to extend the "structure" and "behavior" section a bit.